src/exampleWorker.js

```diff
@@ -0,0 +1,229 @@
+/* eslint-env browser */
+
+'use strict';
+
+self.importScripts('../dist/ohm.js', 'utils.js');
+
+// Web Worker that generates examples.
+// communicates with 'exampleGenerationUI.js'
+
+(function(root, initModule) {
+  if (typeof exports === 'object') {
+    module.exports = initModule;
+  } else {
+    initModule(root.ohm, root.utils);
+  }
+})(this, function(ohm, utils) {
```

Comment:
I think you should change everything above this comment to the following:

```js
/* eslint-env browser */

'use strict';

// Web Worker that generates examples.
// communicates with 'exampleGenerationUI.js'

(function(root, initModule) {
  if (typeof exports === 'object') {
    module.exports = initModule;
  } else {
    initModule(root.ohm, root.utils, self);
  }
})(this, function(ohm, utils, workerGlobalScope) {
  var self = workerGlobalScope;

  self.importScripts('../dist/ohm.js', 'utils.js');
```

Basically, just move the `importScripts` call inside the main function body, and add an extra arg to the main function that allows the `self` to be stubbed out.

That change would make this a valid Node module (right now it throws a ReferenceError), and is a good start at making it testable.